recipes/patsy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.4.1" %}
+{% set gh_org = "pydata" %}
+{% set gh_repo = "patsy" %}
+{% set pkg_name = "patsy" %}
```

Comment:
Again, leftover from some of my earlier work where I was hacking a number recipes at one go... it basically boiled down to a few archetypes, and these values were all that had to change between them. Feel free to take them out!

I do foresee us eventually being able to make use of the higher-level data with something like [jinja2schema](https://github.com/aromanovich/jinja2schema), using the named variables as a higher-level representation of the metadata, but this is probably further out!